components/01-globals/typography/heading/_heading.scss

```diff
@@ -1,16 +1,16 @@
-$headings-margin                : 0 0 0.5em 0 !default;
-$headings-font-family           : inherit !default;
-$headings-font-weight           : 500 !default;
-$headings-line-height           : 1.1 !default;
-$headings-color                 : inherit !default;
-$headings-text-transform        : none !default;
+$headings-margin        : 0 0 0.5em 0 !default;
+$headings-font-family   : $font-family-base-bold !default;
+$headings-font-weight   : 900 !default;
+$headings-line-height   : 1.1 !default;
```

Comment:
shouldn't we use our default line-height from project here? Why that one should be smaller?

Reply:
My bad, done :)